crypto/gaes/gaes.go

```diff
@@ -80,38 +81,38 @@ func DecryptCBC(cipherText []byte, key []byte, iv ...[]byte) ([]byte, error) {
 	blockModel := cipher.NewCBCDecrypter(block, ivValue)
 	plainText := make([]byte, len(cipherText))
 	blockModel.CryptBlocks(plainText, cipherText)
-	plainText, e := PKCS5UnPadding(plainText, blockSize)
+	plainText, e := PKCS7UnPadding(plainText, blockSize)
 	if e != nil {
-		return nil, e
+		return nil, gerror.NewCode(gcode.CodeInvalidParameter, e.Error())
 	}
 	return plainText, nil
 }
 
-func PKCS5Padding(src []byte, blockSize int) []byte {
+func PKCS7Padding(src []byte, blockSize int) []byte {
 	padding := blockSize - len(src)%blockSize
 	padtext := bytes.Repeat([]byte{byte(padding)}, padding)
 	return append(src, padtext...)
 }
 
-func PKCS5UnPadding(src []byte, blockSize int) ([]byte, error) {
+func PKCS7UnPadding(src []byte, blockSize int) ([]byte, error) {
 	length := len(src)
 	if blockSize <= 0 {
-		return nil, gerror.NewCode(gcode.CodeInvalidParameter, "invalid blocklen")
+		return nil, errors.New("invalid blocklen")
```

Comment:
This updates make error stack information loosing when calling this function.

Reply:
Yep

Reply:
This needs modification